Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG ENV
 RUN apt-get -y update && apt-get -y --no-install-recommends install \
         make gcc libc-dev libgeos-dev musl-dev libpq-dev libffi-dev
 
-RUN pip install --upgrade pip && pip install pipenv==2021.5.29
+RUN pip install --upgrade pip && pip install pipenv
```

Comment:
Do you happen to remember why this was originally constrained? 🤷

Reply:
Pipenv, as useful as it is, seems to take "move fast and break things" as its mantra. Without pinning, we frequently have deployments start failing. This seems to be a comment on a previous commit in which I removed the pin, but I subsequently re-pinned to v2022.11.30 which I have found to be reliable (and use in a number of other repos).